trailblazer/server/api.py

```diff
@@ -257,6 +257,28 @@ def put_set_analysis_uploaded():
         store.set_analysis_uploaded(
             case_id=content.get("case_id"), uploaded_at=content.get("uploaded_at")
         )
-        return jsonify("Success! Update request sent"), 201
+        return jsonify("Success! Uploaded at request sent"), 201
+    except Exception as error:
+        return jsonify(f"Exception: {error}"), 409
+
+
+@blueprint.route("/set-analysis-status", methods=["PUT"])
+def put_set_analysis_status():
+    content: Response.json = request.json
+
+    try:
+        store.set_analysis_status(case_id=content.get("case_id"), status=content.get("status"))
+        return jsonify("Success! Analysis set to fail request sent"), 201
+    except Exception as error:
+        return jsonify(f"Exception: {error}"), 409
+
+
+@blueprint.route("/add-comment", methods=["PUT"])
+def put_add_comment():
```

Comment:
Is the put necessary in the function name? Not sure what the best practice is.

Reply:
I use PUT when updating an existing object and POST when adding/creating a new object.